docs/design/da-pki-revocation-design.md

```diff
@@ -0,0 +1,213 @@
+# DA PKI Revocation
+
+## User Stories
+
+1. Revocation of PAI
+
+   A Vendor with DCL write privilege can submit a transaction to publish the location of the CRL distribution point for a PAA associated with the Vendor ID. This information can be updated by the Vendor.
+
+2. Revocation of a DAC 
+
+   A Vendor with DCL write privilege can submit a transaction to publish the location of the CRL distribution point for a PAI associated with the Vendor ID. This information can be updated by the Vendor.
+
+
+## Revocation Distribution Point Schema
+The schema is the same as in `PR6360`.
+However, the validation logic is proposed to be a bit different (see [Validation Logic](#validation-logic) and [Divergence](#divergence-from-pr6360)). 
+
+
+| Name                 | Type    | Constraint          | Mutable | Conformance |
+|:---------------------|:--------|:--------------------|:--------|-------------|
+| VendorID             | uint16  | all                 | No      | M           |
+| ProductID            | uint16* | all                 | No      | O (desc)    |
+| IsPAA                | bool    | all                 | No      | M           |
+| Label                | string  | max 64              | No      | M           |
+| CRLSignerCertificate | string  | max 2048            | No      | M           |
+| IssuerSubjectKeyID   | string  | max 64              | No      | M           |
```

Comment:
This may need to become mutable, and some checks done on transcations where an update is done. Would that work?

Reply:
What is the use case to mutate it?
I thought that we have an assumption that `IssuerSubjectKeyID` is equal to the CRLSignerCertificate's Subject Key Identifier. What are the cases when they are different?